evaluation/data/ims/C1152/ims_regrid.nml

```diff
@@ -0,0 +1,8 @@
+&ims_regrid_nml
+ obs_source="IMS4km"
+ fv3_grid="C1152"
+ land_static_filename="/scratch2/NCEPDEV/land/data/forcing/era5/static/C1152/ufs-land_C1152_hr3_static_fields.nc"
```

Comment:
all these static file locations have changed, do you think we should change them now before the merge?

Reply:
I just ls them, and they are still there. You can merge them now if you do not remove these static file right now. If we update them future, it should update mx resolution as wells. What do you think?